plasmapy/particles/tests/test_particle_class.py

```diff
@@ -1518,3 +1515,23 @@ def test_molecule_other():
         assert CustomParticle(2 * 126.90447 * u.u, e.si, "I2 1+") == molecule(
             "I2 1+", Z=1
         )
+
+def test_undefined_charge():
+    """Test that a particle with an undefined charge retruns |nan| C"""
+    H_particle = Particle("H")
+    assert u.isclose(H_particle.charge, np.nan * u.C, equal_nan=True)
+
+def test_undefined_standard_atomic_weight():
+    """Test that a particle with an undefined standard atomic weight retruns |nan| kg"""
+    Pm_particle = Particle("Pm")
+    assert u.isclose(Pm_particle.standard_atomic_weight, np.nan * u.kg, equal_nan=True)
+
+def test_undefined_mass():
+    """Test that a particle with an undefined mass retruns |nan| kg"""
+    tau_neutrino_particle = Particle("tau neutrino")
+    assert u.isclose(tau_neutrino_particle.mass, np.nan * u.kg, equal_nan=True)
+
+def test_undefined_mass_energy():
+    """Test that a particle with an undefined mass energy retruns |nan| J"""
+    nu_tau_particle = Particle("nu_tau")
+    assert u.isclose(nu_tau_particle.mass_energy, np.nan * u.J, equal_nan=True)
```

Comment:
```suggestion
def test_undefined_charge():
    """Test that a particle with an undefined charge returns |nan| C."""
    H_particle = Particle("H")
    assert u.isclose(H_particle.charge, np.nan * u.C, equal_nan=True)

def test_undefined_standard_atomic_weight():
    """Test that a particle with an undefined standard atomic weight returns |nan| kg."""
    Pm_particle = Particle("Pm")
    assert u.isclose(Pm_particle.standard_atomic_weight, np.nan * u.kg, equal_nan=True)

def test_undefined_mass():
    """Test that a particle with an undefined mass returns |nan| kg."""
    tau_neutrino_particle = Particle("tau neutrino")
    assert u.isclose(tau_neutrino_particle.mass, np.nan * u.kg, equal_nan=True)

def test_undefined_mass_energy():
    """Test that a particle with an undefined mass energy returns |nan| J."""
    nu_tau_particle = Particle("nu_tau")
    assert u.isclose(nu_tau_particle.mass_energy, np.nan * u.J, equal_nan=True)

```
...and so that the file ends with a blank line.  That could also be done by commenting `pre-commit.ci autofix` as a comment in this pull request.